.mergify.yml

```diff
@@ -162,7 +162,7 @@ pull_request_rules:
   - name: backport patches to 7.17 branch
     conditions:
       - merged
-      - label=backport-v7.17.0
+      - label ~= ^(backport-v7.17.0|backport-7.17)$
```

Comment:
IIUC, https://docs.mergify.com/configuration/conditions/#operators-on-lists should work in this case

Reply:
```suggestion
      - label~=^(backport-v7.17.0|backport-7.17)$
```